src/attestation.go

```diff
@@ -87,3 +87,16 @@ func arePCRsIdentical(ourPCRs, theirPCRs map[uint][]byte) bool {
 	}
 	return true
 }
+
+func isEnclaveInDebugMode(doc *nitrite.Document) bool {
+	pcr0, exists := doc.PCRs[0]
```

Comment:
Do you happen to remember if all PCRs are zero (like the [AWS docs suggest](https://docs.aws.amazon.com/enclaves/latest/user/set-up-attestation.html#attestation-doc)) or just PCR0? I currently don't have access to a machine to test this myself.

Reply:
I'll test it today

Reply:
Sorry for the delay, in my testing only PCR4 was non-zero.

Reply:
Gotcha! This should be good to merge then.